recipes/json2html/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install 
```

Comment:
Delete this file and use the following instead (from the example recipe):

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26